src/main/java/com/sap/sgs/phosphor/fosstars/tool/github/MarkdownReporter.java

```diff
@@ -98,7 +99,7 @@ public void runFor(List<GitHubProject> projects) throws IOException {
     for (GitHubProject project : allProjects) {
       stars.put(project, starsOf(project));
     }
-    allProjects.sort(Comparator.comparingInt(stars::get));
+    allProjects.sort(Collections.reverseOrder(Comparator.comparingInt(stars::get)));
```

Comment:
allProjects.sort(Comparator.comparingInt(stars::get, Comparator.reverseOrder())); would this be better? 

- I feel like Collections.reverseOrder() would have O(n^2) and Comparator.comparingInt() another O(n^2). Maybe I am wrong.

Reply:
> * I feel like Collections.reverseOrder() would have O(n^2) and Comparator.comparingInt() another O(n^2). Maybe I am wrong.

I am not sure about that. I think `Collections.reverseOrder()` just inverse the comparator. In other words, just changes the sign of value returned by the wrapped comparator. That should not increase complexity.

Reply:
ok. good to know.